pontoon/base/static/js/translate.js

```diff
@@ -1529,6 +1525,19 @@ var Pontoon = (function (my) {
             e.stopPropagation();
           }
         });
+
+        $('p#sign-in-required > a#sidebar-signin').click(function (e) {
+          e.preventDefault();
+
+          if($(this).is("#sidebar-signin")) {
+            self.startLoader();
+            django_browserid.login().then(function(verifyResult) {
+              window.location.reload();
+            }, function(jqXHR) {
+              self.endLoader('Oops, something went wrong.', 'error');
+            });
+          }
+        })
```

Comment:
You should move this inside [django_browserid.registerWatchHandlers()](http://django-browserid.readthedocs.org/en/latest/api/javascript.html#registerWatchHandlers). You should also put lines 1533-1538 in a private function and reuse it in both Sign in triggers.